snooty.toml

```diff
@@ -1,9 +1,11 @@
 name = "node"
 title = "Node.js"
-
 toc_landing_pages = ["/fundamentals/authentication", "/fundamentals", "/fundamentals/crud", "/usage-examples"]
 
 [constants]
-version = 4.0
+version = 3.6
 package-name-org = "mongodb-org"
 pgp-version = "{+version+}"
+api = "https://mongodb.github.io/node-mongodb-native/{+version+}"
+mongosh = "``mongosh``"
+driver = "node"
```

Comment:
Comment: As these source constants are not used, should these be removed? This is sort of related to this [PR discussion around constants](https://github.com/mongodb/docs-visual-studio-extension/pull/2#discussion_r761268859) (may fall under YAGNI).

Reply:
Sounds good - removing them. Thanks for the context 👍

Reply:
This needs to be quoted.